README.md

```diff
@@ -209,9 +210,10 @@ spec:
   retrycount: 5
   retryinterval: "5s"
   timeout: "60s"
+  credentialparameters:
+    --password
   sharedparameters:
     --username: "admin"
-    --password: "password"
```

Comment:
Out of date

Reply:
IIUC you are referring to ` --password: "password"` which is already removed in this PR